### PR TITLE
Add customisable container name

### PR DESCRIPTION
**Pull request by TimvdHeijden**
# In this PR
- Make it possible to insert a custom name for the container
- Create the container if it does not exist

### CODE REVIEW

File: df_to_azure/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from .export import df_to_azure
 
-__version__ = "0.7.0-rc.1"
+__version__ = "0.8.0-rc.1"
```
Comment (erfannariman) [inline df_to_azure/__init__.py:5]: Eventueel release `0.7.0-rc.2` van maken? Aangezien `0.7.0` nooit gereleased is.

File: df_to_azure/export.py [hunk 2]
```diff
@@ -32,10 +33,18 @@ def df_to_azure(
...
 
     if parquet:
-        DfToParquet(df=df, tablename=tablename, folder=schema, method=method, id_field=id_field).run()
+        DfToParquet(
+            df=df,
+            tablename=tablename,
+            folder=schema,
+            method=method,
+            id_field=id_field,
+            container_parquet=container_parquet,
```
Comment (erfannariman) [inline df_to_azure/export.py:46]: Kunnen we het `container_name` noemen ipv `container_parquet`? Wat denk jij?

**Review by erfannariman — CHANGES_REQUESTED** [review]
Twee punten om te bespreken.

File: df_to_azure/export.py (hunk 2 above)
Comment (TimvdHeijden) [reply df_to_azure/export.py:46]: That's ok, we will alway use containers for parquet, so we do not need to clarify it in the variable name.

File: df_to_azure/__init__.py (hunk 1 above)
Comment (TimvdHeijden) [reply df_to_azure/__init__.py:5]: Yes, I also still saw an open PR related to version number so was not sure what to do with this.

**Pull request merged**